client/components/data/page/index.jsx

```diff
@@ -0,0 +1,98 @@
+/**
+ * External dependencies.
+ */
+import { Component, PropTypes } from 'react';
+import { connect } from 'react-redux';
+import { bindActionCreators } from 'redux';
```

Comment:
You know, I saw that, read it, understood it, and then ended up totally ignoring it. :)